src/Microsoft.Data.SqlClient/tests/StressTests/SqlClient.Stress.Framework/StressConfigReader.cs

```diff
@@ -0,0 +1,98 @@
+                    string sourceName = sourceNavigator.GetAttribute("name", nsUri);
+                    string sourceType = sourceNavigator.GetAttribute("type", nsUri);
+                    bool isDefault;
+                    isDefault = bool.TryParse(sourceNavigator.GetAttribute("isDefault", nsUri), out isDefault) ? isDefault : false;
+                    string dataSource = sourceNavigator.GetAttribute("dataSource", nsUri);
+                    string user = sourceNavigator.GetAttribute("user", nsUri);
+                    string password = sourceNavigator.GetAttribute("password", nsUri);
+                    string database = sourceNavigator.GetAttribute("database", nsUri);
+                    bool supportsWindowsAuthentication;
+                    supportsWindowsAuthentication = bool.TryParse(sourceNavigator.GetAttribute("supportsWindowsAuthentication", nsUri), out supportsWindowsAuthentication) ? supportsWindowsAuthentication : false;
+                    bool isLocal;
+                    isLocal = bool.TryParse(sourceNavigator.GetAttribute("isLocal", nsUri), out isLocal) ? isLocal : false;
+                    bool disableMultiSubnetFailover;
+                    disableMultiSubnetFailover = bool.TryParse(sourceNavigator.GetAttribute("disableMultiSubnetFailover", nsUri), out disableMultiSubnetFailover) ? disableMultiSubnetFailover : false;
+                    bool disableNamedPipes;
+                    disableMultiSubnetFailover = bool.TryParse(sourceNavigator.GetAttribute("disableNamedPipes", nsUri), out disableNamedPipes) ? disableNamedPipes : false;
+                    bool encrypt;
+                    encrypt = bool.TryParse(sourceNavigator.GetAttribute("encrypt", nsUri), out encrypt) ? encrypt : false;
+
+                    DataSourceElement element = new DataSourceElement(sourceName, sourceType, null, dataSource, database, user, password, ds_isDefault: isDefault, ds_isLocal: isLocal, disableMultiSubnetFailoverSetup: disableMultiSubnetFailover, disableNamedPipes: disableNamedPipes, encrypt: encrypt);
+                    Sources.Add(element);
+                }
+            }
+            catch (XmlException e)
+            {
+                throw new InvalidDataException($"Error reading configuration file '{_configFilePath}': {e.Message}", e);
+            }
+            catch (IOException e)
+            {
+                throw new InvalidDataException($"Error reading configuration file '{_configFilePath}': {e.Message}", e);
+            }
+            catch (System.Exception e)
+            {
+                throw new InvalidDataException($"Error reading configuration file '{_configFilePath}': {e.Message}", e);
+            }
+            finally
+            {
+                reader?.Dispose();
+            }
+        }
+
+        private XmlReader CreateReader()
+        {
+            FileStream configurationStream = new FileStream("SqlClient.Stress.Framework/" + _configFilePath, FileMode.Open);
```

Comment:
I had to remove the prefix to get this to run:

```suggestion
            FileStream configurationStream = new FileStream(_configFilePath, FileMode.Open);
```